gqcp/tests/QCMethod/CC/CCD0_test.cpp

```diff
@@ -0,0 +1,257 @@
+#define BOOST_TEST_MODULE "CCD0"
+
+#include <boost/test/unit_test.hpp>
+#include <iostream>
+#include <fstream>
+#include <string>
+
+#include "QCMethod/CC/CCD0.hpp"
+#include "Basis/ScalarBasis/GTOBasisSet.hpp"
+#include "Basis/ScalarBasis/GTOShell.hpp"
+#include "Basis/SpinorBasis/RSpinorBasis.hpp"
+#include "Basis/SpinorBasis/USpinorBasis.hpp"
+#include "Molecule/Molecule.hpp"
+#include "Operator/SecondQuantized/SQTwoElectronOperator.hpp"
+#include "Operator/SecondQuantized/SQHamiltonian.hpp"
+#include "Mathematical/Representation/QCRankFourTensor.hpp"
+#include "Operator/FirstQuantized/NuclearRepulsionOperator.hpp"
+#include "Operator/FirstQuantized/OverlapOperator.hpp"
+#include "Operator/SecondQuantized/SQOneElectronOperator.hpp"
+#include "Operator/SecondQuantized/SQTwoElectronOperator.hpp"
+#include "QCMethod/HF/RHFSCFEnvironment.hpp"
+#include "QCMethod/HF/RHFSCFSolver.hpp"
+#include "QCMethod/HF/DiagonalRHFFockMatrixObjective.hpp"
+#include "QCMethod/HF/RHF.hpp"
+#include "QCMethod/QCStructure.hpp"
+#include "Mathematical/Representation/BlockRankFourTensor.hpp"
+
+GQCP::QCRankFourTensor<double> twoElectronAntiSymmetrized(const GQCP::QCRankFourTensor<double>& gs) {
```

Comment:
We often name the operators `g_op` and then we use the parameters as `const auto& g_par = g_op.parameters()`.

Reply:
Are you saying that the `twoElectronAntiSymmetrized` parameter should be a `SQTwoElectronOperator` instead of a `QCRankFourTensor` such that I can define `g_par` within the function?

Reply:
Meaning that the return type of the function should be a `SQTwoElectronOperator` too instead of a `QCRankFourTensor`?

Reply:
Yes, that's something I would suggest.